README.md

```diff
@@ -348,10 +331,7 @@ be changed to latest.
 
 ####`package_gpg_key`
 
-RPM package GPG key to import. Uses source method. Should be a URL for Debian/RedHat
-OS family, or a file name for RedHat OS family.
-Set to http://www.rabbitmq.com/rabbitmq-signing-key-public.asc by default.
-Note, that `key_content`, if specified, would override this parameter for Debian OS family.
+This should generally be left as default, unless you know what you're doing.
```

Comment:
Sorry, removed this in the first commit, then added it back in with a slightly more descriptive note. I'm not actually sure that it's that useful except maybe in the case where the package is coming from a third party repo configured by the user that's not using the signing key... I think when the vendor package is used from the upstream OS, the key should already be imported, so I'm guessing not many cases where you'd want to change this. Alternately, we could just make it non-configurable and remove the docs?

Using the packagecloud repo is odd because the metadata is signed by packagecloud's key, but the package itself is signed by RMQ's signing key, which I think is the only reason this is necessary.